data/json/recipes/recipe_food.json

```diff
@@ -3049,6 +3056,75 @@
   ]
 },{
   "type" : "recipe",
+  "result": "brew_milk_curdled",
+  "result_mult" : 3,
```

Comment:
Is it supposed to be this low? 15 milks into only 3 curdled? Cheese takes 6 curdled (30 milk total) for 3 portions, making cheesemaking very expensive. And that's even before counting the stomachs/herbs.